src/Test/Unit/Config/SchemaTest.php

```diff
@@ -127,6 +128,7 @@ public function testGetDefaultsForPostDeploy(): void
                     '',
                 ],
                 PostDeployInterface::VAR_TTFB_TESTED_PAGES => [],
+                PostDeployInterface::VAR_VERBOSE_COMMANDS => '',
```

Comment:
I've seen this fix in two other PRs already. One of these needs to be merged ASAP

Reply:
This should also include fix for travis.yml